documentation/development-processes-checklists.md

```diff
@@ -21,51 +23,61 @@ Each sprint is considered to progress through these three phases. Further detail
 
 The checklists below must be completed at the beginning of the phase for which they are named.
 
-_Note: Except on the first merge of a bugfix/feature/hotfix branch, please always use `git merge branchName` with up-to-date local branches, and not `git pull origin branchName`, as the latter forces a non-FF merge, which is often undesirable._
+_Note: Except on the first merge of a hotfix/bugfix/feature branch, please always use `git merge branchName` with up-to-date local branches, and not `git pull origin branchName`, as the latter forces a non-FF merge, which is often undesirable._
 
 In total, each sprint cycle will include the following actions in the following order:
 
-1. Complete "Before Initialization Phase Checklist".
-1. Complete "Sprint Closeout Procedures".
-1. Complete "Before Development Phase Checklist".
-1. Proceed with development in accordance with the "Development Procedures".
-1. Complete "Before Testing Phase Checklist".
-1. Complete "Release Procedures".
+1. [Initialization Phase Checklist](#initialization-phase-checklist)
+1. [Sprint Closeout Procedures](#sprint-closeout-procedure)
+1. [Before Development Phase Checklist](#before-development-phase-checklist)
+1. Proceed with development in accordance with the [Development Procedures](#development-procedures)
+1. [Testing Phase Checklist](#testing-phase-checklist)
+1. [Release Procedure](#release-procedure)
```

Comment:
they are (or should be) in this order.  My thought here is that this also serves as a table of contents.  Those are page links there, so information is a little easier to get to quickly.